hack/verify-bazel.sh

```diff
@@ -43,9 +44,21 @@ kazel_diff=$("${TMP_GOPATH}/bin/kazel" \
   -root="${TESTINFRA_ROOT}")
 
 if [[ -n "${gazelle_diff}" || -n "${kazel_diff}" ]]; then
-  echo "${gazelle_diff}"
-  echo "${kazel_diff}"
-  echo
-  echo "Run ./hack/update-bazel.sh"
+  echo "${gazelle_diff}" >&2
+  echo "${kazel_diff}" >&2
+  echo >&2
+  echo "Run ./hack/update-bazel.sh" >&2
+  exit 1
+fi
+
+# Make sure there are no BUILD files outside vendor - we should only have
```

Comment:
hmm, any reason to leave these in vendor? 🤔

Reply:
projects we vendor (such as the other kubernetes repos) might still be using `BUILD` instead of `BUILD.bazel`, and I'm not sure it makes sense to move them.

which actually makes me realize now that running `hack/update-deps.sh` is causing a bunch of diffs, due to filenames changing. let me fix that...